validation-test/radar/r118770461.swift

```diff
@@ -0,0 +1,8 @@
+@usableFromInline
```

Comment:
A couple of notes here:

- I think it would be better to move this to test/Concurrency and use `issue70019.swift` as a name because it’s publicly reported.
- This test currently does nothing because it lacks a `// RUN:` command, please take a look how other tests in test/Concurrency set that up with swift-frontend invocation.

Reply:
Sorry I'm new here, but but could tell me does there any document for swift-frontend invocation? I wonder what's the meaning of the texts after `// RUN:` and what command should I fill in it.

Reply:
I’d need to take a look regarding documentation but would be able to help you out with RUN command regardless if you want. Meanwhile you can try copying from test/Concurrency/sendable_*.swift

Reply:
I think Pavel meant `test/Concurrency`, not `validation-test/Concurrency`. I agree this sort of simple test case fits better in the primary test suite. Funnily enough, I have been around contributing for quite some time, and the validation test suite is *such a mixup* that I am still ignorant to its original purpose. I personally make it as a dumping ground for:

* Internal tooling tests
* Complex non-contrived Swift source code that cannot be reduced to focus on a particular code path

Reply:
> I wonder what's the meaning of the texts after `// RUN:` 

These are shell commands that define the test together with the remaining contents of the file. Running the test means running all the `RUN:` commands in the file in order. 

> and what command should I fill in it.

https://github.com/apple/swift/blob/main/docs/Testing.md#writing-tests

Reply:
Thank you for the doc link, @AnthonyLatsis!

Reply:
Thanks! @AnthonyLatsis